backend/jupiterbrain.go

```diff
@@ -339,7 +367,12 @@ func (p *jupiterBrainProvider) Start(ctx gocontext.Context, startAttributes *Sta
 		}
 		instance.Stop(ctx)
 
-		return nil, ctx.Err()
+		select {
+		case err := <-errChan:
+			return nil, err
+		case <-time.After(2 * time.Second):
```

Comment:
Again with pushing such values to config.  I realize this means more overhead in the form of config parsing, struct members, and argument passing.  I'd much rather have the option to alter a runtime config than have to wait for a PR->build->binary cycle.